Marlin/src/pins/stm32f4/pins_NEPTUNE_3.h

```diff
@@ -0,0 +1,139 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (c) 2021 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ *
+ * Based on Sprinter and grbl.
+ * Copyright (c) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+#pragma once
+
+//
+// MKS Robin E3 V2 pinmap
+//
+
+// Avoid conflict with TIMER_TONE
+#include "env_validate.h"
+//
+#define ALLOW_STM32F4
+
+// Release PB4 (Z_DIR_PIN) from JTAG NRST role
+//
+// #define DISABLE_DEBUG
+
+#define BOARD_INFO_NAME "MKS Robin Nano E3"

```

Comment:
Yep, this is mistake, not nano e3 indeed

Reply:
That was provided to me by elegoo, they based it on the nano e3 but it is custom

Reply:
I just updated the name to fix the confusion